addons/mail/static/src/models/popover_view/popover_view.js

```diff
@@ -17,13 +17,83 @@ registerModel({
         },
     },
     recordMethods: {
+        /**
+         * Returns whether the given html element is inside the component
+         * of this popover view.
+         *
+         * @param {Element} element
+         * @returns {boolean}
+         */
+        contains(element) {
+            if (this.component && this.component.el.contains(element)) {
+                return true;
+            }
+            return false;
```

Comment:
maybe just `return Boolean(this.component && this.component.el.contains(element));`